tests/unit/pages/Editor.spec.js

```diff
@@ -1047,6 +1047,42 @@ describe("Editor.vue", () => {
     expect(wrapper.vm.itemImage).toBe(imageURL);
   });
 
+  it("checks whether subjective warning is shown when survey question", async () => {
+    const isSurveyQuestion = jest.spyOn(Editor.computed, "isSurveyQuestion");
+    wrapper = mount(Editor);
+
+    const dummyItemDetailsWithSurvey = clonedeep(global.dummyItemDetails);
+
+    await wrapper.setData({
+      items: clonedeep(global.dummyItems),
+      itemDetails: clonedeep(global.dummyItemDetails),
+      currentItemIndex: 2,
+    });
+
+    expect(isSurveyQuestion).toHaveBeenCalled();
```

Comment:
`toHaveBeenCalled` check is applied for methods and not for computed properties.